packages/x-components/src/views/accessibility-check.vue

```diff
@@ -1,118 +1,46 @@
 <template>
```

Comment:
Don't you want to keep the previous elements? We can build a sub-router if you want to have your own a11y playgrounds. It can be helpful to have safe places to check all of these behaviors.

Reply:
sounds good